src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -641,9 +665,16 @@ void Snippet::SnippetJitExecutor::generate(const jit_snippets_compile_args* jcp)
     SNIPPETS_REGISTER_PASS_RELATIVE(Place::After, ov::intel_cpu::pass::FuseLoadStoreConvert,
                                     ov::intel_cpu::pass::SetBrgemmCopyBBuffersShape);
 
-    schedule = snippetAttrs.snippet->generate_from_linear_ir(std::make_shared<ov::snippets::lowered::pass::PassConfig>(),
+    auto lowering_config = std::make_shared<ov::snippets::lowered::pass::PassConfig>();
+#ifdef SNIPPETS_LIBXSMM_TPP
+    lowering_config->disable<ov::snippets::lowered::pass::OptimizeDomain>();
```

Comment:
Could you please comment out here why we disable this optimization for tpp?
If I got it correctly, the reason is that we transform ops to TPP opset and set port descriptors on data flow.
Probably we should control these subtensors in this optimization? I mean `OptimizeDomain` should check the body that there are not set subtensors. I'm just thinking loud :) Seems like `OptimizeDomain` pass is not safe in general)) And we should cover this case in the near future as the separate ticket.
What do you think?